src/util.js

```diff
@@ -76,17 +77,121 @@ function getIssueInfo(issue) {
  */
 function makeCSV(issues, file) {
   try {
-    const fields = ['repositoryUrl', 'title', 'body', 'labels'];
+    const fields = ['text', 'labels'];
     const json2csvParser = new Json2csvParser({fields, unwind: 'labels'});
     const csv = json2csvParser.parse(issues);
     fs.appendFileSync(file, csv);
   } catch (error) {
-    log.warn('ERROR WRITING ISSUES DATA TO CSV.');
+    log.error('ERROR WRITING ISSUES DATA TO CSV.');
+  }
+}
+
+/**
+ * Create a Google AutoML Natural Language dataset
+ * @param {string} projectId
+ * @param {string} computeRegion
+ * @param {string} datasetName
+ * @param {string} multiLabel
+ */
+async function createDataset(
+  projectId,
+  computeRegion,
+  datasetName,
+  multilabel
+) {
+  // [START automl_natural_language_createDataset]
```

Comment:
We don't actually need these unless you intend to write a tutorial on cloud.google.com and show these code samples?